cmd/db.go

```diff
@@ -0,0 +1,95 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	db "github.com/cosmos/cosmos-db"
+)
+
+const (
+	rocksDBMagic  = "\xf7\xcf\xf4\x85\xb7\x41\xe2\x88"
+	pebbleDBMagic = "\xf0\x9f\xaa\xb3\xf0\x9f\xaa\xb3" // 🪳🪳
+	levelDBMagic  = "\x57\xfb\x80\x8b\x24\x75\x47\xdb"
+	magicLen      = 8
+)
+
+func GetFormat(path string) (db.BackendType, error) {
+	magic, err := getMagic(path)
+	if err != nil {
+		return "invalid", err
+	}
+	switch string(magic) {
+	case rocksDBMagic:
+		// Bera returns rocksdb but it's pebble
+		return db.PebbleDBBackend, nil
```

Comment:
feel like we'll shoot ourselves into the foot once we want to support rocksdb, or once we hit a rocksdb instance

so this behaviour feels like something implementation specific, and I wonder if we can make it chain-specific somehow

NB. I remember we tried replacing goleveldb with rocksdb once (for Sei) but IIRC we didn't see much performance boost, and also it was inferior to pebbledb. Aside from that I don't remember ever seeing rocksdb